charts/4allportal/templates/webdav/configmap.yaml

```diff
@@ -124,4 +127,63 @@ data:
     BrowserMatch " Konqueror/4" redirect-carefully
     BrowserMatch "^gvfs" redirect-carefully
     BrowserMatch "WebDAVLib" redirect-carefully
+  log-handler.sh: |
+    #!/bin/ash
+    ENDPOINT="http://{{ include "common.names.fullname" . }}-backend/api/extensions/dam/fs-event"
+    while IFS= read -r line; do
+      status=$(echo "$line" | cut -d';' -f1)
+      type=$(echo "$line" | cut -d';' -f2)
+      path=$(echo "$line" | cut -d';' -f3)
+      dest=$(echo "$line" | cut -d';' -f4 | awk '{
```

Comment:
I am not 100% sure where this happens for $path but when moving a file that has '"' in its file-name, they get escaped correctly. This does not happen in $dest.

See attached screenshots for details
<img width="1469" alt="path variable" src="https://github.com/4ALLPORTAL/helm-charts/assets/113340892/099a9574-f3a8-4968-813d-ae1cd0c6e341">
<img width="1875" alt="dest variable" src="https://github.com/4ALLPORTAL/helm-charts/assets/113340892/8b7c95b5-29d2-45e2-b37a-6d64dc1eeb12">

Reply:
$path processed inside WebDAV, but $dest comes as complete link (that's why I've added awk script to unencode it). Will also add step for quotes in it.

Reply:
Done